### PR TITLE
Add `starlite` to the doc navigation list

### PR DESCRIPTION
**Pull request by dhirschfeld**
## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Documentation

## Issues Fixed or Closed by This PR

*

## Checklist

- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the CONTRIBUTING document.
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by dhirschfeld** [comment]
Resolves #3081

File: docs/README.md [hunk 1]
```diff
@@ -80,6 +80,7 @@
 - [Sanic](./integrations/sanic.md)
 - [Chalice](./integrations/chalice.md)
 - [Starlette](./integrations/starlette.md)
+- [Starlite](./integrations/starlite.md)
```
Comment (patrick91) [inline docs/README.md:83]: I think it might be worth renaming the page and link to litestar, since they changed name 😊

**Comment by dhirschfeld** [comment]
Seems to work :tada:

image

File: docs/README.md (hunk 1 above)
Comment (dhirschfeld) [reply docs/README.md:83]: Sure, can do...

File: docs/integrations/litestar.md [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 
-See the example below for integrating Starlite with Strawberry:
+See the example below for integrating Litestar with Strawberry:
 
 ```python
 import strawberry
-from starlite import Starlite
-from strawberry.starlite import make_graphql_controller
+from litestar import Litestar
+from strawberry.litestar import make_graphql_controller
```
Comment (patrick91) [inline docs/integrations/litestar.md:17]: oh, we haven't updated the code yet, if you wanna do that to we should do an alias from the old name and do a deprecation, I'm also fine with just updating the docs for now

Comment (dhirschfeld) [reply docs/integrations/litestar.md:17]: I've got to switch to $dayjob now. I  might be able to get to that this weekend but I'm happy for you to add the necessary changes here or merge this and add the changes in a follow-up PR.

**Pull request closed without merging**